src/clj/game/cards/hardware.clj

```diff
@@ -444,17 +441,15 @@
              {:req (req (and (first-event? state :runner :successful-run)
                              (pos? (count-virus-programs state))))
               :optional
-              {:prompt "Place a virus counter?"
-               :yes-ability
-               {:delayed-completion true
-                :effect (effect (continue-ability
-                                  {:prompt "Select an installed virus program"
-                                   :choices {:req #(and (installed? %)
-                                                        (has-subtype? % "Virus")
-                                                        (is-type? % "Program"))}
-                                   :msg (msg "place 1 virus counter on " (:title target))
-                                   :effect (effect (add-counter target :virus 1))}
-                                  card nil))}}}}}
+                   {:prompt "Place a virus counter?"
+                    :yes-ability {:effect (effect (continue-ability
+                                                    {:prompt "Select an installed virus program"
```

Comment:
Can this be pulled down one level directly into the `yes-ability`? So get rid of the continue-ability?

Reply:
Did you miss a commit?